script/ghIssues.pl

```diff
@@ -561,8 +564,11 @@ sub add_developer {
     try{
         if($user){
             my $ddgc_name = $user->username;
+            
             # Give edit permissions to the contributor
-            $ia->add_to_users($user);
+            my $ia = $d->rs('InstantAnswer')->find({id => $ia_hash->{id}});
+            $ia->add_to_users($user) unless ($ia->users->find($user->id) || $user->admin);
```

Comment:
Cool thanks,  Can you check that `$ia` exists here before using it?  Now I'm just trying to be cautious.